azure-pipelines.yml

```diff
@@ -54,14 +61,14 @@ jobs:
     name: Internet_Explorer
     vmImage: 'vs2017-win2016'
     nodeVersion: '12.x'
-    testCommand: 'npm run test:internet-explorer -- -- --ci'
+    browser: 'internet-explorer'
```

Comment:
Do you think we can remove this browser from the list?

Reply:
I thinks this still can be beneficial for someone. Still many developers have to support IE. Maybe next year if Edge became more stable.